md/dates-management-tutorial.md

```diff
@@ -0,0 +1,67 @@
+# How to work with dates in forms and BDM
```

Comment:
I don't think an additional tutorial is needed, but maybe a section in this one can be added.